.circleci/config.yml

```diff
@@ -368,6 +379,11 @@ workflows:
           requires:
             - build_www
           context: build_www
+      - sync-i18n-source:
```

Comment:
This runs whenever the master branch is updated right? (I can't tell since this file is so big and git cuts it off.

Would it be possible to just run it like... when something in `docs/` changes?

Reply:
Yup, this would only be run on commits to `master` (this is set on line 386)

Circle doesn't natively support checking commits for changes to a directory but we could do something like `git diff HEAD^ HEAD docs/` to check if the last commit included changes to `docs` and exit early accordingly